2.4/test/run

```diff
@@ -145,11 +145,34 @@ function test_config_option() {
     docker stop $(get_cid ${name})
 }
 
+function test_text_search() {
```

Comment:
```bash
# need to set these because `mongo_cmd` relies on global variables
USER=admin
PASS=adminPassword
DB=admin

test_connection ${name}
mongo_local_cmd ${name} 'admin --quiet --eval "JSON.stringify(db.runCommand({getParameter: 1, "textSearchEnabled":1}))"' | grep '"textSearchEnabled":true'
```